### ISSUE TITLE
Toolbox 1.8.1c fails to install in Mac OSX 10.10.5

### ISSUE THREAD

**Issue by datmark** [docker/toolbox#138]
I am having issues trying to get install Toolbox 1.8.1c.

I get this summary at the end of the installation:
![screen shot 2015-09-01 at 9 42 56 am](https://cloud.githubusercontent.com/assets/1057340/9607359/53d5a98a-508e-11e5-9333-42183adeecb2.png)

.. and when I run it on the command line

```bash
installer: Package name is Docker Toolbox
installer: Upgrading at base path /
installer: The upgrade failed (The Installer encountered an error that caused the installation to fail. Contact the software manufacturer for assistance.)
```

and I get this when running the Docker Terminal...

```bash
open /Users/user/.docker/machine/machines/default/config.json: permission denied
docker is configured to use the default machine with IP 
For help getting started, check out the docs at https://docs.docker.com

open /Users/user/.docker/machine/machines/default/config.json: permission denied
```

and running the script found in:
```bash
/Applications/Docker/Docker Quickstart Terminal.app/Contents/Resources/Scripts
```
... completes but i get:
```bash


                        ##         .
                  ## ## ##        ==
               ## ## ## ## ##    ===
           /"""""""""""""""""\___/ ===
      ~~~ {~~ ~~~~ ~~~ ~~~~ ~~~ ~ /  ===- ~~~
           \______ o           __/
             \    \         __/
              \____\_______/


docker is configured to use the default machine with IP 192.168.99.100
For help getting started, check out the docs at https://docs.docker.com

$ docker info
Get http:///var/run/docker.sock/v1.20/info: dial unix /var/run/docker.sock: no such file or directory.
* Are you trying to connect to a TLS-enabled daemon without TLS?
* Is your docker daemon up and running?
```

I am not sure what else I could do... please help!

**Comment by v1k0d3n** [comment]
<pre>
sudo docker info 
</pre>

should produce better results ;)

looks like docker is running as root (when it's installed) which actually concerns me a little more. i changed the directory of ~./docker to my user and the installer changed it back to root. see #140.

**Comment by v1k0d3n** [comment]
try the work-around in #140 and let me know if it works for you (until docker updates or provides more information on the installer).

**Comment by datmark** [comment]
Unfortunately, the workaround you provided in #140 did not work for my situation, which is weird...

```bash
$ docker-machine create -d virtualbox default
Creating CA: /Users/user/.docker/machine/certs/ca.pem
Creating client certificate: /Users/user/.docker/machine/certs/cert.pem
Image cache does not exist, creating it at /Users/p235969/.docker/machine/cache...
No default boot2docker iso found locally, downloading the latest release...
Downloading https://github.com/boot2docker/boot2docker/releases/download/v1.8.1/boot2docker.iso to /Users/user/.docker/machine/cache/boot2docker.iso...
Creating VirtualBox VM...
Creating SSH key...
Starting VirtualBox VM...
Starting VM...
Error creating machine: Maximum number of retries (60) exceeded
You will want to check the provider to make sure the machine and associated resources were properly removed.
```

but

![screen shot 2015-09-02 at 5 24 42 pm](https://cloud.githubusercontent.com/assets/1057340/9646257/8b36d796-5197-11e5-968e-a348c95a3c74.png)
and
![screen shot 2015-09-02 at 5 26 06 pm](https://cloud.githubusercontent.com/assets/1057340/9646273/bd9c5c92-5197-11e5-8397-16c1470effbb.png)

I hope these helps someone...

**Comment by datmark** [comment]
After so many installs/uninstalls/deleting folders I was finally able to successfully install docker (and the toolbox).

How I got it to work this time...
1. Deleted anything in the system that mentions docker.
2. Installed toolbox 1.8.1c.
3. Ran the Docker Quickstart Terminal (which failed).
4. Deleted ~/.docker
5. Installed docker using
```bash
$ docker-machine create -d virtualbox default
```

I am not sure how it started working again...
